src/components/utils/TextUtils.ts

```diff
@@ -1,7 +1,7 @@
 /**
```

Comment:
You can / should remove `/**` JSDoc comment if they are no longer useful (like in this case).

Reply:
Done!